src/router/storeSync/CrossHairParamConfig.class.js

```diff
@@ -99,6 +77,21 @@ function generateCrossHairUrlParamFromStoreValues(store) {
     return null
 }
 
+function acceptedValues(store, query) {
```

Comment:
The name of the functions implies that it would return a list and not a boolean

Reply:
it seems to me that it is more a validate function, so `validate` seems better.

Reply:
I must admit it started as a list, and then thought it would be better for it to be a boolean directly, but forgot to change the name. This is a good suggestion.